src/test/ui/async-await/issue-102138.rs

```diff
@@ -0,0 +1,46 @@
+// check-pass
```

Comment:
It looks like there are two copies of the test.

Reply:
Deleted.